node/cn/backend.go

```diff
@@ -329,6 +329,11 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 		reward.NewStakingManager(cn.blockchain, governance, cn.chainDB)
 	}
 
+	// Call CreateSnapshot to restore the votes since last checkpoint
+	if err := cn.engine.CreateSnapshot(cn.blockchain, cn.blockchain.CurrentBlock().NumberU64(), cn.blockchain.CurrentBlock().Hash(), nil); err != nil {
```

Comment:
Can we change the place of the added code line? 
It is a little strange to me to create a governance snapshot and update govParam here. 
Because we already load the latest gov param at the above code lines, and the restored governance data is used to configure `blockchain`. If we update the governance-related data here again, then it can be misunderstood like "the governance data above was not correct"

https://github.com/klaytn/klaytn/blob/f3605ce7f24131ca23e89772497bad5a6740f71b/node/cn/backend.go#L270-L285

Reply:
perhaps, restoring changeSet could be simple and clear

Reply:
@aidan-kwon At first I thought it would be easy to restore a changeSet, but my thought wrong. There are a lot of things to consider. `apply` in `Snapshot` function already considered all the things. I moved the UpdateParam outside of the createSnapshot.

So, calling `CreateSnapshot` at start point of the node makes sense to me in next two aspects
* restore the snapshots of the blocks from latestGovernanceState to currentBlock
* restore the in-memory gov states (e.g. gov.changeSet) (not gov.currentSet.. its restoration place is initializeCache)

Can I go with the original code?